src/lib/page_scripts/market_listing.ts

```diff
@@ -1,7 +1,11 @@
 import {init} from './utils';
 import '../components/market/item_row_wrapper';
 import '../components/market/utility_belt';
+import '../components/market/response_render_results_status';
+import {initResponseRenderResultsStatus} from '../components/market/response_render_results_status';
 
 init('src/lib/page_scripts/market_listing.js', main);
 
-async function main() {}
+async function main() {
+    initResponseRenderResultsStatus()
```

Comment:
As mentioned in another comment, it's preferrable for a decorator of the component to handle it's own injection into the page, so that you don't need to handle creation manually here.

Then you just need to import the file.